script.js

```diff
@@ -1,4 +1,309 @@
+      playerLastWinState = 1;
+      winnerState = 1;
+      return `축하합니다! ${userName} Won! You chose ${user} while Computer chose ${com}! <br>Enter another input to Muk Jji Paa!`;
+    }
+    return winMessage(user, com, symbolGenerator(user), symbolGenerator(com));
+  }
+  // If user chose paper and com is scissors, user lose
+  // If user chose scissors and com is stone, user lose
+  // If user chose stone and com is paper, user lose
+  if (
+    (user == "paper" && com == "scissors") ||
+    (user == "scissors" && com == "stone") ||
+    (user == "stone" && com == "paper")
+  ) {
+    //Play korean mode if gameMode is korean
+    if (gameMode == "korean") {
+      //Update computer winning status and record winning position
+      comLastWinState = 1;
+      winnerState = 1;
+      return `죄송합니다! Com Won! You chose ${user} while Computer chose ${com}!<br>Enter another input to fight back!`;
+    }
+    return loseMessage(user, com, symbolGenerator(user), symbolGenerator(com));
+  }
+  //else it's a draw
+  else {
+    //Play korean mode if gameMode is korean
+    if (gameMode == "korean") {
+      //Both condition true and draw is obtained, player wins
+      if (playerLastWinState == 1 && winnerState == 1) {
+        //Reset states
+        winnerState = 0;
+        playerLastWinState = 0;
+
+        return (
+          winMessage(user, com, symbolGenerator(user), symbolGenerator(com)) +
+          `<br>${winLossPerc(
+            playerScore,
+            comScore
+          )} Play Again?<br>Current Game Mode:${gameMode}`
+        );
+      }
+      //Both condition true and draw is obtained, computer wins
+      if (comLastWinState == 1 && winnerState == 1) {
+        //Reset states
+        winnerState = 0;
+        comLastWinState = 0;
+        return (
+          loseMessage(user, com, symbolGenerator(user), symbolGenerator(com)) +
+          `<br>${winLossPerc(
+            playerScore,
+            comScore
+          )} Play Again?<br>Current Game Mode:${gameMode}`
+        );
+      }
+      return `It's a draw! Try Again! Muk Jji Paa!`;
+    }
+    return drawMessage(user, com, symbolGenerator(user), symbolGenerator(com));
+  }
+}
+
+// Reverse SPS Game function
+function reverseGame(user, com) {
+  // If user chose paper and com is scissors, user wins
+  // If user chose scissors and com is stone, user wins
+  // If user chose stone and com is paper, user wins
+  if (
+    (user == "paper" && com == "scissors") ||
+    (user == "scissors" && com == "stone") ||
+    (user == "stone" && com == "paper")
+  ) {
+    return winMessage(user, com, symbolGenerator(user), symbolGenerator(com));
+  }
+  // If user chose stone and com is scissors, user lose
+  // If user chose paper and com is stone, user lose
+  // If user chose scissors and com is paper, user lose
+  if (
+    (user == "stone" && com == "scissors") ||
+    (user == "paper" && com == "stone") ||
+    (user == "scissors" && com == "paper")
```

Comment:
Same here, to reduce repeats of user win or use lose, can be refactor into functions:
```
checkuserWin()
checkUserLose()
```
But your comments are clear enough for me! Keep it up!